content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,87 @@ ifdef::backend-html5[]
\ No newline at end of file
+= Administering Jenkins on Kubernetes
+
+In this section, we will look at:
+
+* Upgrading from one version of Kubernetes to another.
+* Kubernetes Concepts.
+
+== Upgrading from one version of Kubernetes to another:
+
+Kubernetes (K8s) is an open-source system for automating deployment, scaling, and management of containerized applications.
+
+A Kubernetes cluster adds a new automation layer to Jenkins.
+Kubernetes makes sure that resources are used effectively and that your servers and underlying infrastructure are not overloaded.
+Kubernetes' ability to orchestrate container deployment ensures that Jenkins always has the right amount of resources available.
+
+=== How to upgrade the version of Kubernetes in your cluster:
+
+Upgrading your Kubernetes cluster is one of the tasks you must take on as a Jenkins/Kubernetes administrator.
+Kubernetes delivers link:https://kubernetes.io/releases[new releases] regularly, and will only maintain releases for the three latest versions.
+Therefore, if something breaks or there is a security issue on an older version, Kubernetes will not fix it, but ask you to upgrade.
+All the major cloud providers only support a given set of latest versions.
+
+We aim to support the same set of versions supported by the major cloud providers, Google, Amazon and Azure.
+These versions are described in the links below, ranging from version 1.20 to version 1.24.
+
+* link:https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html[Amazon]
+* link:https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar[Azure]
+* link:https://cloud.google.com/kubernetes-engine/docs/release-notes[Google]
+
+*Upgrade Procedure:*
+
+It is highly recommended to test upgrades on a separate cluster, where you have the same environment used in production.
+
+* Run link:https://github.com/FairwindsOps/pluto[Pluto] to find any deprecated CRDs present in your cluster.
+You can run Pluto on the cluster using 'pluto detect-helm' and on the files on your cluster repository using 'pluto detect-files'.
+
+* Cross-check the list of deprecations you find using the list from link:https://kubernetes.io/docs/reference/using-api/deprecation-guide[Kubernetes], and see what is required for the Kubernetes version you want to upgrade to.
+
+* Fix what needs to be fixed, and deploy the changes.
+
+You may now start the upgrade of Kubernetes.
+The process has two steps: 
+. Upgrade the control plane.
+. Upgrade the nodes. 
+
+Follow the upgrade steps according to your cloud provider.
+* link:https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[Amazon]
+* link:https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli[Azure]
+* link:https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster[Google]
+* Pour yourself a cup of your favorite drink while you sit back and wait for the magic to happen.
```

Comment:
```suggestion
Follow the upgrade steps according to your cloud provider.

* link:https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html[Amazon]
* link:https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli[Azure]
* link:https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster[Google]

Pour yourself a cup of your favorite drink while you sit back and wait for the magic to happen!
```
Same as above for this list, adding in a blank line between 63 & 64 to separate that step from the upgrade guides